tools/apigee-sackmesser/cmd/report/opdk/apiproducts.sh

```diff
@@ -0,0 +1,38 @@
+echo "<h3>API Products</h3>" >> "$report_html"
+
+mkdir -p "$export_folder/$organization/config/resources/edge/env/$environment/api-products"
+
+sackmesser list "organizations/$organization/apiproducts"| jq -r -c '.[]|.' | while read -r apiProductName; do
+        sackmesser list "organizations/$organization/apiproducts/$apiProductName" > "$export_folder/$organization/config/resources/edge/env/$environment/api-products/$apiProductName".json
+        elem_count=$(jq '.entries? | length' "$export_folder/$organization/config/resources/edge/env/$environment/api-products/$apiProductName".json)
+    done
+
+if ls "$export_folder/$organization/config/resources/edge/env/$environment/api-products"/*.json 1> /dev/null 2>&1; then
+    jq -n '[inputs]' "$export_folder/$organization/config/resources/edge/env/$environment/api-products"/*.json > "$export_folder/$organization/config/resources/edge/env/$environment/api-products".json
+fi
+
+echo "<div><table id=\"ts-lint\" data-toggle=\"table\" class=\"table\">" >> "$report_html"
+echo "<thead class=\"thead-dark\"><tr>" >> "$report_html"
+echo "<th data-sortable=\"true\" data-field=\"id\">Product Name</th>" >> "$report_html"
+echo "<th data-sortable=\"true\" data-field=\"envs\">Environments</th>" >> "$report_html"
+echo "<th data-sortable=\"true\" data-field=\"proxies\">Proxies</th>" >> "$report_html"
+echo "<th data-sortable=\"true\" data-field=\"enabled\">Aproval Type</th>" >> "$report_html"
+echo "</tr></thead>" >> "$report_html"
+
+echo "<tbody class=\"mdc-data-table__content\">" >> "$report_html"
+
+jq -c '.[]' "$export_folder/$organization/config/resources/edge/env/$environment/api-products".json | while read i; do 
```

Comment:
Need to validate if the JSON body is having the same structure for OPDK/Edge and X/Hybrid.
Currently, the logic is only being executed for OPDK; not even Edge.